### PR TITLE
update manual output roads parquets -- resolve #76

### LINKED ISSUE #76: [data] ensure correct `output` from #71
[data] ensure correct `output` from #71

* xref https://github.com/uscuni/simplification/issues/70#issuecomment-2186461796

### PR DESCRIPTION
**Pull request by jGaboardi**
This PR resolves #76

### CODE REVIEW

**Comment by martinfleis** [comment]
@dancejod or @Kryndlea, can you verify that versions of parquet files in this branch are all indeed the simplified ones you consider final?

File: data/4617/manual/4617.parquet
Comment (dancejod) [inline data/4617/manual/4617.parquet:1]: ok

File: data/809/manual/809.parquet
Comment (dancejod) [inline data/809/manual/809.parquet:1]: ok

File: data/4617/manual/4617.parquet
Comment (dancejod) [inline data/4617/manual/4617.parquet:1]: ok

File: data/809/manual/809.parquet
Comment (dancejod) [inline data/809/manual/809.parquet:1]: ok

File: data/869/manual/869.parquet
Comment (dancejod) [inline data/869/manual/869.parquet:1]: ok

File: data/1133/manual/1133.parquet
Comment (dancejod) [inline data/1133/manual/1133.parquet:1]: accidentally uploaded the old output version, i've committed the new correct parquet

**Review by dancejod — APPROVED** [review]
I've somehow uploaded the old version of 1133, the correct version is there now. Besides that, all files are OK.

**Comment by martinfleis** [comment]
@jGaboardi I think that @dancejod committed  the correct one to main rather that to this branch. Can you just resolve conflict by keeping main then?

**Pull request merged**